### DISCUSSION TITLE
What about some Discord server for DNGrep?

### DISCUSSION THREAD

**Discussion by paoscripts** [dnGrep/dnGrep#940] [General]
Dear comunity! What about creating a Discord server for DNGrep? To be honest almost all projects noways have such a central hub for our interactions, discussions, announcements, and maybe some spaces to build the friendly comunity or just have fun.
Why Discord?
The interactions are live and faster, we can set up various channels to organize discussions, announcements and support channels, we could even use voice chats to hangouts or q&a sessions. 
What do you think?

**Comment by doug24** [comment]
I'm not sure how many people get notified of discussions here, and there typically is very low traffic. I typically create a discussion topic for each release, but there is almost never any response on them. I will add a link to this discussion in the release notes for next week's release and see if you get some responses.

**Comment by DerellLicht** [comment]
and frankly, discord servers typically really suck, especially if there *is* alot of traffic... all the discussions are dumped together in one long stream, with no connectivity or anything... I've generally been finding them useless... I'd rather have a managed, well-supported forum such as this one.
